### PR TITLE
feat: contest modal

### LINKED ISSUE #87: Contest button and modal
screen shot 2019-01-30 at 11 05 53

Depends on #17

### PR DESCRIPTION
**Pull request by nicosantangelo**
screen shot 2019-02-05 at 11 21 34

Closes #87

### CODE REVIEW

File: src/components/Modals/ContestModal/ContestModal.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+export default class ContestModal extends React.PureComponent<Props> {
+  state = { hasAcceptedTerms: false }
+
+  handleClose = () => {
+    const { modal, onClose } = this.props
+    onClose(modal.name)
+  }
+
+  handleSubmit = () => {
+    console.log('Submit form')
```
Comment (cazala) [inline src/components/Modals/ContestModal/ContestModal.tsx]: 🤸🏻‍♀️

File: src/components/Modals/ContestModal/ContestModal.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      <Modal
+        open={modal.open}
+        className="ContestModal"
+        size="small"
+        onClose={this.handleClose}
+        closeIcon={<CloseModalIcon onClick={this.handleClose} />}
+      >
+        <Modal.Header>
+          <Header size="huge" className="hero">
+            Participate in our contest
```
Comment (cazala) [inline src/components/Modals/ContestModal/ContestModal.tsx]: missing i18n

File: src/components/Modals/ContestModal/ContestModal.tsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+          </Header>
+        </Modal.Header>
+        <Modal.Content>
+          <div className="modal-row">
+            Get in our contest and win LAND and a date with Fede!
```
Comment (cazala) [inline src/components/Modals/ContestModal/ContestModal.tsx]: 😆 🥇

File: src/components/Modals/ContestModal/ContestModal.tsx [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+          <div className="header-image" />
+          <Header size="huge" className="hero">
+            {t('contest_modal.title')}
+          </Header>
+        </Modal.Header>
+        <Modal.Content>
+          <div className="modal-row">
+            <p className="explanation">
+              {t('contest_modal.start')}
+              <br />
```
Comment (belohlavek) [inline src/components/Modals/ContestModal/ContestModal.tsx]: https://media.giphy.com/media/PJoLp4gDIqjYs/giphy.gif

**Pull request merged**